plugins/europeana/entity.js

```diff
@@ -118,14 +118,23 @@ export function getEntitySlug(entity) {
  * @return {Object} related entities
  */
 export function relatedEntities(type, id, params) {
+  let apiParams = {
```

Comment:
Changing the relatedEntities logic shouldn't be in scope for this.
The related entities logic is known to not be very useful and the mechanism is likely going to be replaced, so we shouldn't attempt to add additional logic that takes into account the theme param.

Reply:
Fair comment. I have removed the logic for applying theme for related entities. I have however kept the change to use `entityQuery` to keep consistent with that used for the entity search results if it has no theme set. From my testing a few entities, this does not affect the related entitities returned.